templates/zerver/invite_user.html

```diff
@@ -32,7 +32,12 @@ <h3 id="invite-user-label">{% trans %}Invite users to Zulip{% endtrans %}</h3>
                 {% endif %}
                 <div class="control-group">
                     <label class="control-label" for="streams_to_add">{{ _('Streams they should join') }}</label>
-                    <div class="controls" id="streams_to_add"></div>
+                    <div class="controls">
+                        <div id="default_stream_message">
+                            New members will automatically join <span id="default_streams"></span> <a href="#" id="edit_streams_button">Edit</a>
```

Comment:
* Lets just list the streams here. No need of `New members will automatically join` as it already has a label `Streams they should join`. 
* Try creating 7 or 8 streams and add it as default streams. All the streams are not visible. 
* I don't think bold styling of stream names is necessary.